build/NuSpecs/WindowsAppSDK-Nuget-Native.targets

```diff
@@ -16,6 +16,12 @@
     </Reference>
   </ItemGroup>
 
+  <ItemGroup Condition="'$(AppxPackage)' == 'true'">
```

Comment:
I'm not sure whey we don't just remove the "Condition" from the above entry for Microsoft.Windows.ApplicationModel.DynamicDependency.winmd?

Reply:
To avoid any further unforeseen issues in unconditionally adding the 'implementation' parameter as well in the reference I did like this. WinUI was just adding 'private' for the references with its wildcard, so just replicating the same here.